src/hdskins/java/com/voxelmodpack/hdskins/HDSkinManager.java

```diff
@@ -311,14 +308,18 @@ public ResourceLocation getConvertedSkin(ResourceLocation res) {
         return loc == null ? res : loc;
     }
 
+    public void convertSkin(BufferedImage image) {
+        Graphics graphics = image.getGraphics();
+        convertSkin(image, graphics);
+        graphics.dispose();
+    }
+
     public void convertSkin(BufferedImage image, Graphics dest) {
-        for (ISkinModifier skin : skinModifiers) {
```

Comment:
I don't disagree with this, but not gui related.

Reply:
To be honest as this point I don't even care. The first two or three commits exist purely so I can read this code and not gouge my own eyes out.